src/RuleSet/DeclarationBlock.php

```diff
@@ -58,9 +69,31 @@ public static function parse(ParserState $parserState, ?CSSList $list = null): ?
                             }
                         }
                         break;
+                    case '(':
+                        if (!isset($stringWrapperCharacter)) {
+                            ++$functionNestingLevel;
+                        }
+                        break;
+                    case ')':
+                        if (!isset($stringWrapperCharacter)) {
+                            if ($functionNestingLevel <= 0) {
+                                throw new UnexpectedTokenException('anything but', ')');
+                            }
+                            --$functionNestingLevel;
+                        }
+                        break;
+                    case ',':
+                        if (!isset($stringWrapperCharacter) && $functionNestingLevel === 0) {
+                            $selectors[] = \implode('', $selectorParts);
+                            $selectorParts = [];
+                            $parserState->consume(1);
+                            $consumedNextCharacter = true;
```

Comment:
Unless I'm mistaken, `$consumedNextCharacter` is never read after line 49, and hence we can remove this line and line 59 (where it is written).